GOVERNANCE.md

```diff
@@ -0,0 +1,88 @@
+# etcd Governance
+
+## Principles
+
+The etcd community adheres to the following principles:
+
+- Open: etcd is open source.
+- Welcoming and respectful: See [Code of Conduct](code-of-conduct.md).
+- Transparent and accessible: Changes to the etcd code repository and CNCF related
+activities (e.g. level, involvement, etc) are done in public.
+- Merit: Ideas and contributions are accepted according to their technical merit for
+the betterment of the project. For specific guidance on practical contribution steps
+please see [CONTRIBUTING](./CONTRIBUTING.md) guide.
+
+## Maintainers
+
+Maintainers are first and foremost contributors that have shown they are committed to
+the long term success of a project. Maintainership is about building trust with the
+current maintainers of the project and being a person that they can depend on to make
+decisions in the best interest of the project in a consistent manner. The current
+maintainers are listed in the [MAINTAINERS](./MAINTAINERS) file. The maintainers role
+can be a top-level or restricted to certain package/feature depending upon their
+commitment in fulfilling the expected responsibilities as explained below.
+
+### Top-level maintainer
+
+- Running the etcd release processes
+- Ownership of test and debug infrastructure
+- Triage GitHub issues to keep the issue count low (goal: under 100)
+- Regularly review GitHub pull requests across all pkgs
+- Providing cross pkg design review
+- Monitor email aliases
+- Participate when called upon in the [security disclosure and release process](security/README.md)
+- General project maintenance
+
+### Package/feature maintainer
+
+- Ownership of test and debug failures in a pkg/feature
+- Resolution of bugs triaged to a package/feature
+- Regularly review pull requests to the pkg subsystem
+
+Contributors who are interested in becoming a maintainer, if performing these
+responsibilities, should discuss their interest with the existing maintainers. New
+maintainers must be nominated by an existing maintainer and must be elected by a
+supermajority of  maintainers. Likewise, maintainers can be removed by a supermajority
+of the maintainers and moved to emeritus status.
+
+Life priorities, interests, and passions can change. If a maintainer needs to step
+down, inform other maintainers about this intention, and if possible, help find someone
+to pick up the related work. At the very least, ensure the related work can be continued.
+Afterward, create a pull request to remove yourself from the [MAINTAINERS](./MAINTAINERS)
+file.
+
+## Approvers
```

Comment:
Are approvers and reviewers refer to the list in OWNERS file?

Reply:
@jingyih thank you and yes, that's true. Good idea to provide link to the OWNERS file while mentioning them if that's what you are thinking! Also, most other projects usually have one file - MAINTAINERS or OWNERS - with maintainers/approvers/reviewers listed separately underneath. So I am up for merging them like moving maintainers in the OWNER file or other way.

Reply:
I agree, having both files is also confusing to me, which is why I asked the question initially.

Reply:
Yeah, merging into one sound better.

Reply:
OK, sounds great!